queue/src/fiskaltrust.Middleware.Storage.SQLite/DatabaseInitialization/DatabaseMigrator.cs

```diff
@@ -69,11 +69,27 @@ public DatabaseMigrator(ISqliteConnectionFactory connectionFactory, int timeoutS
             {
                 _logger.LogInformation($"{notAppliedMigrations.Count()} pending database updates were detected. Updating database now.");
             }
+            Exception exception = null;
             foreach (var migrationScript in notAppliedMigrations)
             {
                 _logger.LogDebug($"Updating database with migration script {migrationScript}..");
-                await connection.ExecuteAsync(File.ReadAllText(migrationScript), commandTimeout: _timeoutSec).ConfigureAwait(false);
-                await SetCurrentVersionAsync(connection, Path.GetFileNameWithoutExtension(migrationScript)).ConfigureAwait(false);
+                try
+                {
+                    await connection.ExecuteAsync(File.ReadAllText(migrationScript), commandTimeout: _timeoutSec).ConfigureAwait(false);
+                    await SetCurrentVersionAsync(connection, Path.GetFileNameWithoutExtension(migrationScript)).ConfigureAwait(false);
+                }
+                catch (Exception ex)
+                {
+                    //Bug Fix #385, Migrationscript 004 is not executed in Launcher
+                    if (!migrationScript.Contains("014_FailedFinishTransactionRequest.sqlite3") && !ex.Message.Contains("Request"))
+                    {
+                        exception = new Exception(ex.Message);
```

Comment:
maybe just `throw;` here? I don't think we need to store the exception, right? 🤔

Reply:
When the Database was launched with the Testlauncher the 004 script was executed and a exception would throw here, therefor the try catch of this exception